pulsar-broker/src/test/java/com/yahoo/pulsar/websocket/proxy/ProxyPublishConsumeTlsTest.java

```diff
@@ -133,7 +118,7 @@ public void socketTest() throws InterruptedException, NoSuchAlgorithmException,
             Assert.assertTrue(produceSocket.getBuffer().size() > 0);
             Assert.assertEquals(produceSocket.getBuffer(), consumeSocket.getBuffer());
         } catch (Throwable t) {
-            log.error(t.getMessage());
+            Assert.fail(t.getMessage());
```

Comment:
Can we log the full exception trace here before failing the test? It might help in understanding the reason of the exception.